packages/node_modules/@node-red/editor-client/src/js/ui/view.js

```diff
@@ -692,17 +690,21 @@ RED.view = (function() {
                     cancel: function() {
                         quickAddActive = false;
                         resetMouseVars();
+                        updateSelection();
+                        hideDragLines();
+                        redraw();
                     },
-                    add: function(type) {
+                    add: function(type,auto) {
```

Comment:
don't know if there is a better way to tell the add function we want to stay in wiring mode - as we can't use the cmd key being held down like we do when using mouse.